docroot/themes/custom/dcp/sass/components/_sponsors-block.scss

```diff
@@ -99,7 +101,7 @@
       content: "";
       position: absolute;
       left: 0;
-      background: url(/../themes/custom/dcp/images/design2.svg) no-repeat;
+      background: url(/../themes/custom/dcp/images/design2.svg) left -150px no-repeat;
```

Comment:
@vidyanarke - I think to resolve this permanently,  we have to remove the white space from SVG image from the top and bottom.  Else if the number of sponsor increases then again the image position will get changed. You can contact the design team to remove the whitespace part from the SVG image.

Reply:
We will request a new cropped image from the design team, meanwhile, we will merge this and change it later if needed